addons/addon-environment-sc-ui/packages/environment-type-mgmt-ui/src/models/forms/CfnParamsForm.js

```diff
@@ -34,8 +34,22 @@ import { createForm } from '@aws-ee/base-ui/dist/helpers/form';
  */
 function getCfnParamsForm(cfnParams, existingParamValues) {
   const isAppStreamEnabled = process.env.REACT_APP_IS_APP_STREAM_ENABLED === 'true';
+  const filteredCfnParams = cfnParams.filter(cfnParam => {
+    const { ParameterKey } = cfnParam;
+    // We don't need to provide `IsAppStreamEnabled` as a config option to user because we can pull this value from settings on the backend
+    const keysToFilterOut = ['IsAppStreamEnabled'];
+    if (isAppStreamEnabled) {
+      keysToFilterOut.push('AccessFromCIDRBlock');
+    } else {
+      keysToFilterOut.push('EgressStoreIamPolicyDocument');
+      keysToFilterOut.push('SolutionNamespace');
+    }
```

Comment:
I didn't quite get why `EgressStoreIamPolicyDocument ` and `SolutionNamespace` need to be filtered out only if AppStream is disabled.

* Is there an advantage of making `SolutionNamespace` configurable when we are deriving it anyways from stackId.
* `EgressStoreIamPolicyDocument` seems like something that should either not be configurable or only show up when egress config is enabled.

Also update unit tests for this method?

Reply:
Good point. Let's not make `EgressStoreIamPolicyDocument` and `SolutionNamespace` configurable by the customer. 

I'll also update the unit test.